modules/contentbox/models/ui/ThemeService.cfc

```diff
@@ -664,16 +664,32 @@ component accessors="true" threadSafe singleton{
 							<div class="panel-body">
 					' );
 
+					// Show group intro if there is one
+					if( structKeyExists( thisSettingMD, "groupIntro" ) and thisSettingMD.groupIntro gt '' ){
```

Comment:
The default values are all set here: https://github.com/Ortus-Solutions/ContentBox/pull/273/files#diff-433bcbe5d41747870423cd8a8d6d49e4L627

Add all the new field defaults there.  This way there is no need to do structkey exists.